express_webpack/index.html

```diff
@@ -1,9 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
+<!-- This Web Application Manifest is required for iOS 16.4 WebPush -->
+<link rel="manifest" href="manifest.json" />
+
 <!-- Some sites use this HTML base tag to change the origin for all relative links.
     Ensure OneSignal SDK correctly handles this. -->
-<base href="https://www.example.com/">
+<!-- NOTE: This does not work with the relative pathed manifest.json above. -->
+<!--<base href="https://www.example.com/"> -->
```

Comment:
Can we just remove the example.com base tag entirely? I always comment it out anyway since it prints a warning

Reply:
It is commented out by default now in the PR, so think we should keep the comment so there is some awareness as it should be tested in some scenarios.